auditbeat/docs/getting-started.asciidoc

```diff
@@ -189,105 +77,75 @@ auditbeat.modules:
-=== Step 3: Load the index template in {es}
-
-include::{libbeat-dir}/shared-template-load.asciidoc[]
+. From the installation directory, run:
++
+--
+include::{libbeat-dir}/tab-widgets/setup-widget.asciidoc[]
+--
++
+`-e` is optional and sends output to standard error instead of syslog.
 
-[[load-kibana-dashboards]]
-=== Step 4: Set up the {kib} dashboards
+This step loads the recommended index template for writing to {es}
+and deploys the sample dashboards for visualizing the data in {kib}.
 
-include::{libbeat-dir}/dashboards.asciidoc[]
+[TIP]
+=====
+A connection to {es} (or {ess}) is required to set up the initial
+environment. If you're using a different output, such as {ls}, see
+<<load-template-manually>> and <<load-kibana-dashboards>>.
+=====
 
-[id="{beatname_lc}-starting"]
+[float]
+[[start]]
 === Step 5: Start {beatname_uc}
 
-Run {beatname_uc} by issuing the appropriate command for your platform. If you
-are accessing a secured {es} cluster, make sure you've configured credentials as
-described in <<{beatname_lc}-configuration>>.
-
-NOTE: If you use an init.d script to start {beatname_uc} on deb or rpm, you can't
-specify command line flags (see <<command-line-options>>). To specify flags,
-start {beatname_uc} in the foreground.
-
-*deb and rpm:*
+Before starting {beatname_uc}, modify the user credentials in
++{beatname_lc}.yml+ and specify a user who is
+<<privileges-to-publish-events,authorized to publish events>>.
 
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-sudo service {beatname_lc} start
-----------------------------------------------------------------------
+To start {beatname_uc}, run:
 
-*mac and linux:*
+// tag::start-step[]
+include::{libbeat-dir}/tab-widgets/start-widget.asciidoc[]
+// end::start-step[]
 
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-sudo chown root {beatname_lc}.yml <1>
-sudo ./{beatname_lc} -e
-----------------------------------------------------------------------
-<1> To monitor system files, you'll be running {beatname_uc} as root, so you
-need to change ownership of the configuration file, or run {beatname_uc} with
-`--strict.perms=false` specified. See
-{beats-ref}/config-file-permissions.html[Config File Ownership and Permissions]
-in the _Beats Platform Reference_.
+{beatname_uc} should begin streaming events to {es}.
 
 If you see a warning about too many open files, you need to increase the
 `ulimit`. See the <<ulimit,FAQ>> for more details.
 
-include::{libbeat-dir}/shared-brew-run.asciidoc[]
-
-*win:*
-
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-PS C:{backslash}Program Files{backslash}{beatname_uc}> Start-Service {beatname_lc}
-----------------------------------------------------------------------
+//REVIEWERS: Removing cURL test because I think users will see a better view of
```

Comment:
I guess you don't want to merge this :)

Reply:
Ignore this comment if you do :smile:

Reply:
Thanks! I'll remove all the reviewer notes in the next commit.